packages/xchain-thorchain-query/src/thorchain-query.ts

```
@@ -693,272 +692,18 @@ export class ThorchainQuery {
-      toAddress: depositQuote.inbound_address, // Recipient address
-      memo: depositQuote.memo, // Memo
-      estimatedWaitTime: depositQuote.inbound_confirmation_seconds || 0, // Estimated wait time
-      canAddSaver: errors.length === 0, // Can add saver flag
-      slipBasisPoints: depositQuote.fees.slippage_bps, // Slip basis points
-      saverCapFilledPercent, // Saver filled capacity
-      recommendedMinAmountIn: depositQuote.recommended_min_amount_in, // Recommended minimum amount in
-      errors, // Errors
-    }
-    return estimateAddSaver // Return the EstimateAddSaver object
+  public async estimateAddSaver(_addAmount: CryptoAmount<Asset | TokenAsset>): Promise<EstimateAddSaver> {
+    throw new Error('Saver deposit quotes are no longer available. THORNode removed the /quote/saver/deposit endpoint in API v3.15.0.')
```

ESLint

[error] Replace `'Saver·deposit·quotes·are·no·longer·available.·THORNode·removed·the·/quote/saver/deposit·endpoint·in·API·v3.15.0.'` with `⏎······'Saver·deposit·quotes·are·no·longer·available.·THORNode·removed·the·/quote/saver/deposit·endpoint·in·API·v3.15.0.',⏎····`

(prettier/prettier)